### ISSUE TITLE
Proposed contracts for original ledger management

### ISSUE THREAD

**Issue by JeffreyBPetersen** [ga4gh/CGT#2]
Draft here: https://gist.github.com/JeffreyBPetersen/4bc4593974e755ac1bc423315dcfeb5a

Posts to lists could refer to documents via hash or to aggregated lists of hashes. The former favors more comprehensive records while the latter minimizes overhead of blockchain interactions. These approaches can be used simultaneously.

**Comment by rcurrie** [comment]
Closing out now that we're using vanilla public/private key signing in the base CGT database. Managing lists and higher level contracts could be done via separate layer on top of the base CGT database - maybe a proxy that adds to a blockchain in parallel to the base CGT, but that's out of scope for the basic CGT database.
